src/components/merchandise/MerchandiseOrderCollectionBlock.tsx

```diff
@@ -141,7 +141,7 @@ const MerchandiseOrderCard: React.VFC<{
           {orderLog.createdAt && (
             <StyledPurchaseDate className="mb-4 d-flex align-items-center">
               <Icon as={CalendarOIcon} className="mr-2" />
-              {`${moment(orderLog.createdAt).format('YYYY-MM-DD hh:mm')} ${formatMessage(messages.purchase)}`}
+              {`${dayjs(orderLog.createdAt).format('YYYY-MM-DD HH:mm:00')} ${formatMessage(messages.purchase)}`}
```

Comment:
Why add `:00` in the end?

Reply:
I forgot where I saw the suffix  `:00` so I added it without much thought.

I just went to find where I saw it, but it looks like I misread it at that time.

If it's not necessary, I can remove it.

Reply:
I think it's not necessary.

Reply:
OK.
modified.